### DISCUSSION TITLE
Feature request.  Optionally show timestamp of results

### DISCUSSION THREAD

**Discussion by DevinCook** [dnGrep/dnGrep#1118] [Ideas]
I often use dnGrep to find old .cs projects but I want to find the latest project that matched the code specifications I am looking for.

A checkbox to show timestamps may prove helpful to similar users.

**Comment by doug24** [comment]
I assume you are asking to see the last modified time of the file in the results tree?  This has been requested before. The problem is the results tree - I can't create columns of information like in Windows Explorer details view.  It would be possible to add more data to end of the text on the file node of the tree, but that makes it very difficult to read.  

Instead, the file details are shown on a tooltip on each file node in the tree:  file name, file type, encoding, size, and date modified.

If you are not seeing this, check the Options dialog - Search results - Show file info toolitps.

**Reply by DevinCook** [reply]
Thanks, didn't notice that before.

You might consider adding the option to add the timestamp info to the report.
